docs/UserGuide.adoc

```diff
@@ -208,41 +217,45 @@ Format: `slot -start <START_DATE>`
 ==== Adds a duty shift for a doctor: `addshift` ====
 
 Adds a new duty shift for a doctor. +
-Format: `addshift [-id STAFF_REFERENCE_ID] -start <START_TIMING> -end <END_TIMING> [-rec REPEATEDLY] [-num REPEATED_TIMES]`
+If both `-rec <INTERVALS>` and `-num <NUMBER_OF_TIMES>` fields are present, appointments of these `<NUMBER_OF_TIMES>` will be added to the patient
```

Comment:
If both `-rec <INTERVALS>` and `-num <NUMBER_OF_TIMES>` fields are present, duty shifts of these `<NUMBER_OF_TIMES>` will be added to the doctor duty roster in `<INTERVALS>`. Otherwise, only one shift will be added. +